src/ol_dbt/models/marts/mitxpro/marts__mitxpro_ecommerce_productlist.sql

```diff
@@ -35,6 +55,17 @@ with ecommerce_product as (
     order by product_id
 )
 
+, ecommerce_course_to_topics as (
+    select
+        course_to_topics.course_id
+        , array_join(array_distinct(array_agg(coursetopic.coursetopic_name)), ', ') as coursetopic_name
+    from course_to_topics
+    left join coursetopic
+        on course_to_topics.coursetopic_id = coursetopic.coursetopic_id
+    group by course_to_topics.course_id
```

Comment:
```suggestion
    select
        course_to_topics.course_id
        , array_join(array_agg(coursetopic.coursetopic_name order by coursetopic.coursetopic_name), ', ') as coursetopic_names
    from course_to_topics
    inner join coursetopic
        on course_to_topics.coursetopic_id = coursetopic.coursetopic_id
    group by course_to_topics.course_id
```

No need to array_distinct as it's unique on coursetopic_name and course_id, and inner join is proper here

Reply:
I've removed array_distinct and changed that to an inner join. Thanks for all your help Rachel!